### ISSUE TITLE
Incorrect parenthesizing of Derivative

### ISSUE THREAD

**Issue by JSS95** [sympy/sympy#18731]
Here is a simple code:
```python
>>> from sympy import Derivative
>>> from sympy.abc import x
>>> Derivative(-Derivative(x,x),x)
```
LaTeX printing of this gives:

![image](https://user-images.githubusercontent.com/90703/75418761-e298ff00-58f9-11ea-9194-4394b3ced9b3.png)

Which needs to be parenthesized to:

![image](https://user-images.githubusercontent.com/90703/75418840-13793400-58fa-11ea-88ad-8fdec4dd6689.png)

In LaTeX string, the first one (current result) is:
```python
"\frac{d}{dx}-\frac{d}{dx}x"
```

This should be parenthesized to:
```python
"\frac{d}{dx}\left(-\frac{d}{dx}x\right)"
```

note; latex rendered at [quicklatex](https://quicklatex.com/)

**Comment by adarshraj365** [comment]
@JSS95 Yes .
this should be parenthesized .

**Comment by smichr** [comment]
Is anyone else noticing that the "needs to be parenthesized to" form is the same (except for size) as the latex form? Where do you want extra parentheses?

**Comment by mcpl-sympy** [comment]
> Is anyone else noticing that the "needs to be parenthesized to" form is the same (except for size) as the latex form? Where do you want extra parentheses?

It seems that there is a rendering issue.  

In ASCII form, the first formula is:
```
 d       d  
---  -  --- x  
dx      dx  
```

The second one is:  
```
 d   /     d   \
---  | -  --- x |
dx   \    dx   /
```

**Comment by JSS95** [comment]
@smichr 
> Is anyone else noticing that the "needs to be parenthesized to" form is the same (except for size) as the latex form? Where do you want extra parentheses?

It seems that there is a rendering issue.  

In ASCII form, the first formula is:
```
 d       d  
---  -  --- x  
dx      dx  
```

The second one is:  
```
 d   /     d   \
---  | -  --- x |
dx   \    dx   /
```

**Comment by oscargus** [comment]
Fixed by #18740
